cli-toolchain/graphql.md

```diff
@@ -1874,215 +2166,740 @@ type Comment {
 }
 ```
 
-#### Generates
+#### Generates
+
+In order to keep connection queries fast and efficient, the GraphQL transform manages
+global secondary indexes (GSIs) on the generated tables on your behalf. In the future we
+are investigating using adjacency lists along side GSIs for different use cases that are
```

Comment:
I think this is not relevant after the `keyName` change

Reply:
@dabit3 lets remove the "In the future..." statement here. We should just document what we have now.